src/js/tracing/reactnativetracing.ts

```diff
@@ -383,6 +379,15 @@ export class ReactNativeTracing implements Integration {
     );
 
     idleTransaction.registerBeforeFinishCallback((transaction) => {
+      // the problem is that the App start cold/warm by this point is unknown yet, the JS bundle is loaded
+      // before the very first Activity, which is when the Android SDK detects the type of the App start.
+      if (this.options.enableAppStartTracking) {
+        logger.log(
+          `[ReactNativeTracing] thats a test`
+        );
+        void this._instrumentAppStart();
+      }
```

Comment:
moved from `setupOnce` to here, since its not known yet the App start, and here it is